griptape/tools/aws_s3_client/tool.py

```diff
@@ -211,6 +212,42 @@ def upload_content_to_s3(
         except Exception as e:
             return ErrorArtifact(f"error uploading objects to the bucket: {e}")
 
+    @activity(
+        config={
+            "description": "Can be used to download objects from an AWS S3 bucket",
+            "schema": Schema(
+                {
+                    Literal(
+                        "bucket_name",
+                        description="The name of the S3 bucket to download from.",
+                    ): str,
+                    Literal(
+                        "object_keys", description="Object keys to download."
+                    ): [],
+                }
```

Comment:
I was thinking it could support multiple buckets:

```python
Literal(
    "objects",
    ...
): [
    {
        Literal("bucket_name", ...): str,
        Literal("object_key"): str
    }
]
```